### DISCUSSION TITLE
Still not syncing

### DISCUSSION THREAD

**Discussion by JosN1** [abraunegg/onedrive#1803] [Q&A] [answered]
Onedrive version 2.4.14 under Fedora 35/Plasma.

When first installed under Fedora 34, this app worked perfectly. After installing Fedora 35 and reinstalling onedrive, the app failed to sync automatically. I recall reading somewhere that others were having the same experience. Now, the app will sync manually, i.e., by a command on the CLI, but it will still not sync automatically. 

Is this fixable? If so, how? 

[Also--is there a donation page to contribute to development and support efforts? I didn't see one.]

**Comment by abraunegg** [comment, marked as answer]
@JosN1 
> Onedrive version 2.4.14 under Fedora 35/Plasma

This is an unsupported client version - even though this is the *latest* release for Fedora:
* https://koji.fedoraproject.org/koji/packageinfo?packageID=26044
* https://bugzilla.redhat.com/buglist.cgi?quicksearch=onedrive

Your options here are to build your own RPM as per https://github.com/abraunegg/onedrive/blob/master/docs/build-rpm-howto.md

> Now, the app will sync manually, i.e., by a command on the CLI, but it will still not sync automatically.
> 
> Is this fixable? If so, how?

Most likely in your upgrade the systemd services that you had activated are now not active? Refer to https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#running-onedrive-as-a-system-service to re-enable.

> Also--is there a donation page to contribute to development and support efforts? I didn't see one

*Still* waiting on GitHub to review application ..

**Comment by abraunegg** [comment]
@JosN1 
> Also--is there a donation page to contribute to development and support efforts? I didn't see one

GitHub has now approved sponsorship | donations - so this is now possible :)
